modules/ROOT/pages/_partials/task-configure-proxy.adoc

```diff
@@ -16,42 +16,64 @@
 . Select *Mule Gateway* as your runtime.
 . Select *Deploy a proxy application* for *Proxy type*.
 //end::first-steps[]
-//tag::target-type[]
+//tag::target-type-heading[]
 
 [[select_and_configure_target_type]]
 = Select and Configure a Target Type
+//end::target-type-heading[]
+//tag::target-type-number[]
+. Select the *Target type* from the following options:
+//end::target-type-number[]
+//tag::target-type-bullet[]
+Select the *Target type* from the following options:
+//end::target-type-bullet[]
 
-. Select the **Target type** from the following options:
-
-* *CloudHub:* Select this option if you want to use the Mule runtime hosted on the cloud by MuleSoft. The
+//tag::target-type[]
++
+* *CloudHub 2.0*: Select this option if you want to use the Mule runtime hosted by MuleSoft in a container-based cloud infrastructure. The
+CloudHub 2.0 proxy includes a CPU with 0.1 vCores by default.
```

Comment:
@glenn-rodgers-sf in case Unified base pricing is enabled, a mule.micro instance will be used (instead of 0.1 vCores).